styles/components/HeroBanner.module.scss

```diff
@@ -0,0 +1,44 @@
+// left side should have icon and apply button
+// right side should have title, divider, tagline and buttons for social media
+// use grid for layout
+.HeroBanner {
+  @apply grid grid-cols-2 gap-4 p-4;
+  @apply my-8;
+}
+
+.leftColumn {
+  @apply col-span-1 grid place-items-center;
+
+  img {
+    @apply w-1/2 h-auto;
+    @apply object-cover mb-4 rounded-xl;
+    @apply bg-black;
+  }
+}
+
+.rightColumn {
+  @apply col-span-1 grid place-items-center;
+  @apply gap-4;
+
+  .title {
+    @apply text-5xl font-bold text-center;
+    @apply text-color-2;
+  }
+
+  .divider {
```

Comment:
can we get rid of this, and style `hr` globally. lets make use of contextual elements :)

Reply:
Fixed

Reply:
Can we get rid of this now?